README.md

```diff
@@ -47,6 +47,8 @@ it is converted to an attribute set equivalent to `{ root = theArg; }`.
 | `version` | The version of the derivation. |
 | `src` | Used by `naersk` as source input to the derivation. When `root` is not set, `src` is also used to discover the `Cargo.toml` and `Cargo.lock`. |
 | `root` | Used by `naersk` to read the `Cargo.toml` and `Cargo.lock` files. May be different from `src`. When `src` is not set, `root` is (indirectly) used as `src`. |
+| `allRefs` | Whether to fetch all refs while fetching Git dependencies. Useful if the wanted revision isn't in the default branch. |
```

Comment:
Can you add a test for this functionality? See `tests.nix` for existing tests.

Reply:
I've tried to add a test where `Cargo.lock` refers to a repository with non-master-named branch (which, from what I gathered, is one of the use cases for `allRefs`), but everything seems to be working properly even without the switch; do you maybe have an idea on how to create a test for that?